notebooks/tutorials/netcdf-to-cog-cmip6.ipynb

```diff
@@ -0,0 +1,1349 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    client = boto3.client("s3")
</pre><p>Is this line necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-IMPACT/veda-docs/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Well spotted. Removed.